tests/vmi_configuration_test.go

```diff
@@ -2427,11 +2427,11 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 			// ordering:
 			// use a small disk for the other ones
 			containerImage := cd.ContainerDiskFor(cd.ContainerDiskCirros)
-			// virtio - added by NewRandomVMIWithEphemeralDisk
-			vmi = tests.NewRandomVMIWithEphemeralDiskAndUserdata(containerImage, "echo hi!\n")
-			// sata
```

Comment:
So being a little familiar with this test suite (`vmi_configuration_test.go`), these comments are often quite useful. Maybe we could keep it artisanal and not hide behind NewCirros?

Reply:
OK, sice these comments are helpful for some, I'll add them back.

Reply:
I also mean, maybe we should avoid wrappers like NewCirros in here

Reply:
I don't want to duplicate NewCirros. I think the comment does the work

Reply:
I think for this test suite, `libvmi.New` fits better, but as long as there are comments that's fine